u2flib_server/utils.py

```diff
@@ -56,8 +70,15 @@ def sha_256(data):
     return h.digest()
 
 
-Rand.rand_seed(os.urandom(1024))
+def rand_bytes(n_bytes):
+    return os.urandom(n_bytes)
 
 
-def rand_bytes(n_bytes):
-    return Rand.rand_bytes(n_bytes)
+def verify_ecdsa_signature(payload, pubkey, signature):
+    verifier = pubkey.verifier(signature, ec.ECDSA(hashes.SHA256()))
+    verifier.update(payload)
+
+    try:
+        verifier.verify()
+    except InvalidSignature:
+        raise Exception('Attestation signature verification failed!')
```

Comment:
I also don't mind #3. I'll start with that and we can always change it if you guys think one of the other options would be better.